src/main/java/com/microsoft/sqlserver/jdbc/KerbAuthentication.java

```diff
@@ -350,8 +315,8 @@ private String findRealmFromHostname(RealmValidator realmValidator, String hostn
         int index = 0;
         while (index != -1 && index < hostname.length() - 2) {
             String realm = hostname.substring(index);
-            if (authLogger.isLoggable(Level.FINEST)) {
-                authLogger.finest(toString() + " looking up REALM candidate " + realm);
+            if (authLogger.isLoggable(Level.FINER)) {
```

Comment:
Why was this necessary?

Reply:
to make it consistent with the rest of the logging in this file